contracts/src/libraries/Quorum.sol

```diff
@@ -34,11 +34,14 @@ library Quorum {
 
             if (i > 0) {
                 XTypes.SigTuple memory prev = sigs[i - 1];
-                require(sig.validatorAddr != prev.validatorAddr, "OmniPortal: duplicate validator");
-                require(sig.validatorAddr > prev.validatorAddr, "OmniPortal: sigs not sorted");
+                require(sig.validatorAddr != prev.validatorAddr, "Quorum: duplicate validator");
+                require(sig.validatorAddr > prev.validatorAddr, "Quorum: sigs not sorted");
             }
 
-            if (_isValidSig(sig, digest)) votedPower += validators[sig.validatorAddr];
```

Comment:
Decided to error here instead of just passing. I think that's more appropriate. Only valid signatures can be submitted. If any invalid signatures are, we revert.